shell/common/display.h

```diff
@@ -0,0 +1,47 @@
+// Copyright 2013 The Flutter Authors. All rights reserved.
+// Use of this source code is governed by a BSD-style license that can be
+// found in the LICENSE file.
+
+#ifndef FLUTTER_SHELL_COMMON_DISPLAY_H_
+#define FLUTTER_SHELL_COMMON_DISPLAY_H_
+
+namespace flutter {
+
+/// Unique IO per display that is stable until the Flutter application restarts.
+/// See also: `flutter::Display`
+typedef uint64_t DisplayId;
+
+/// Some devices do not support multiple displays, in such cases the embedding
+/// might not provide a device identifier. This value is used as the device id
+/// in such cases. There must only be one screen in such cases.
+static constexpr DisplayId kSingleScreenDeviceId = 0;
+
+/// To be used when the display refresh rate is unknown. This is in frames per
```

Comment:
I don't think it matters what the units are here if the value is unknown.

Reply:
Will remove the unit :-)